src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IVariableDeclaration.vb

```diff
@@ -310,10 +328,10 @@ Module Program
 End Module]]>.Value
 
             Dim expectedOperationTree = <![CDATA[
-IVariableDeclarationStatement (2 variables) (OperationKind.VariableDeclarationStatement) (Syntax: 'Dim i1, i2  ... New Integer')
-  IVariableDeclaration: i1 As System.Int32 (OperationKind.VariableDeclaration) (Syntax: 'i1')
-    Initializer: IObjectCreationExpression (Constructor: Sub System.Int32..ctor()) (OperationKind.ObjectCreationExpression, Type: System.Int32) (Syntax: 'New Integer')
-  IVariableDeclaration: i2 As System.Int32 (OperationKind.VariableDeclaration) (Syntax: 'i2')
+IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclarationStatement) (Syntax: 'Dim i1, i2  ... New Integer')
+  IVariableDeclaration (2 variables) (OperationKind.VariableDeclaration) (Syntax: 'i1, i2 As New Integer')
+    Variables: Local_1: i1 As System.Int32
+      Local_2: i2 As System.Int32
```

Comment:
There are a lot of other tests that depend on this format as well, filed https://github.com/dotnet/roslyn/issues/18752 to track and I'll do this in a follow up PR to avoid large unrelated churn.